modules/graphstudio/pages/load-data.adoc

```diff
@@ -97,18 +97,3 @@ image::Screen Shot 2019-05-16 at 6.28.54 PM.png[]
 
 == Clear Graph Data
```

Comment:
You can delete this part (i think its the title) as well. This functionality is fully moved to the actions page.